src/assets/styles/main.scss

```
@@ -1,4 +1,4 @@
-/*!
+/*
```

Please re-add the explanation
```suggestion
/*!
```